README.md

```diff
@@ -133,7 +137,11 @@ You can set up your repository manually instead of using the automated
     repository using
 
     ~~~
-    $ git push origin gh-pages
+    $ git add index.html
```

Comment:
Replace this block with

    ~~~
    $ git add index.html
    $ git commit -m "updated index.html with helpful information"
    $ git push origin gh-pages
    ~~~

And you can remove the duplicated code block. =)